internal/deviations/deviations.go

```diff
@@ -74,6 +74,10 @@ func P4RTMissingDelete(_ *ondatra.DUTDevice) bool {
 	return *p4rtMissingDelete
 }
 
+func GRIBIIpv4EntryMacOverride(_ *ondatra.DUTDevice) bool {
```

Comment:
Go initialism rules say "Ip"->"IP" and "Mac"->"MAC", so

GRIBIIPv4EntryMACOverride and gribiIPv4EntryMACOverride for the function and flag, respectively

Reply:
Updated them , thanks